### DISCUSSION TITLE
Send multiple pdf file through API endpoint in python

### DISCUSSION THREAD

**Discussion by Luqmanasif1** [FlowiseAI/Flowise#918] [Q&A]
Hi, I hope you are all doing fine. I want to send multiple pdf through the API endpoint in Python. Here is my code but it doesn't pass the PDF file in Flowise, Whenever I pass the PDF through Postman it works fine, I tried a lot to figure out the problem but I can't.

Here is the error that I faced.
SyntaxError: (unicode error) 'Unicode escape' codec can't decode bytes in position 2-3: truncated \UXXXXXXXX escape

A screenshot of my code and postman is given below for your better understanding.

![Capture](https://github.com/FlowiseAI/Flowise/assets/119053488/6a60dee7-b34d-43c6-8957-90b9ee662906)
![Capture1](https://github.com/FlowiseAI/Flowise/assets/119053488/a1cce886-3cb8-41d5-bef3-7dbe387c3988)

**Comment by chungyau97** [comment]
Hi @Luqmanasif1,

You can have quick look on how the code should look like in Python using Postman's Code Snippet.
![image](https://github.com/FlowiseAI/Flowise/assets/33013947/c6e23ff1-e8d5-4ada-9da5-122d4c96f545)

**Comment by mdlfelipe** [comment]
![image](https://github.com/FlowiseAI/Flowise/assets/714543/b0d1c4d4-dc0d-4c96-982f-0bcba960e04b)
I can't point to ask in new document, is possible?
